creator-onboarding/readme/minter-suite.md

```diff
@@ -0,0 +1,69 @@
+---
+order: 851
```

Comment:
@mchrupcala - I don't actually know how the docs site orders pages in the sidbar. I set this to 851, and if you don't know either, I suppose we can see where it ends up 🤷

Reply:
Hmm yeah our ordering in prod looks a bit jumbled (a few pages are all using '500'). Anyway where do you want to add this? Looks like 851 might go between Marketing & Project Pricing but it's hard to tell.
<img width="285" alt="Screen Shot 2023-03-14 at 12 55 49 PM" src="https://user-images.githubusercontent.com/52679312/225080079-672e08c1-ffba-4de4-a64b-2e8aea6a26c2.png">

Reply:
@jakerockland That link makes sense, but not sure why our `project-pricing-model.md` has order 850, but is at the bottom of the nav 😕 🤷 I'm going to merge this with 851 to try and place right above `project-pricing-model.md`, and I'll add a fast-follow if things end up in a poor order 🙏